ghost/i18n/locales/it/portal.json

```diff
@@ -92,16 +92,16 @@
     "If you cancel your subscription now, you will continue to have access until {{periodEnd}}.": "Se annulli ora il tuo abbonamento, continuerai ad avere accesso fino al {{periodEnd}}.",
     "If you have a corporate or government email account, reach out to your IT department and ask them to allow emails to be received from {{senderEmail}}": "Se possiedi un account email aziendale o governativo, contatta il reparto IT e chiedi di consentire la ricezione delle email da parte di {{senderEmail}}",
     "If you would like to start receiving emails again, the best next steps are to check your email address on file for any issues and then click resubscribe on the previous screen.": "Se desideri ricevere nuovamente le email, la cosa migliore da fare è controllare che non ci siano problemi con il tuo indirizzo di posta e poi cliccare su \"Reiscriviti\" nella schermata precedente.",
-    "If you're not receiving the email newsletter you've subscribed to, here are a few things to check.": "Se non ricevi la newsletter a cui sei iscritto, ecco alcuni passaggi da controllare.",
-    "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Se avete completato tutti i passaggi e ancora non ricevete nessuna email, potete contattare l'assistenza a {{supportAddress}}.",
+    "If you're not receiving the email newsletter you've subscribed to, here are a few things to check.": "Se non ricevi la newsletter a cui sei iscritto, ecco alcune cose da controllare.",
+    "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Se avete completato tutti i passaggi e ancora non ricevete nessuna email, potete contattare l’assistenza a {{supportAddress}}.",
```

Comment:
There isn't a particular tone change between "passaggi" and "cose" — even if "cose" is surely used with a more informal tone. The reasoning behind the edit is because in Italian "controllare i passaggi" is a bit of a weird sentence, while you can "completare" — complete — them. "Passaggi" is more akin to "steps," if you will. If you'd prefer, we can make both lines uniform with "cose," though.

Reply:
To clarify - I was asking about formal vs informal 'you'.

Reply:
Of course, you are 100% right. I’ve revised and reworked a little bit the text.